### PR TITLE
Sink exception when fetching latest activities on dashboard

### PR DESCRIPTION
**Pull request by iangledhill**
This sinks exceptions when rendering the latest activities on the dashboard page.

### CODE REVIEW

File: src/SFA.DAS.EAS.Web/Controllers/ActivitiesController.cs [hunk 1]
```diff
@@ -34,10 +35,17 @@ public ActivitiesController(IMapper mapper, IMediator mediator)
...
-            var model = _mapper.Map<LatestActivitiesViewModel>(response);
+            try
+            {
+                var response = _mediator.Send(query);
+                var model = _mapper.Map<LatestActivitiesViewModel>(response);
 
-            return PartialView(model);
+                return PartialView(model);
+            }
+            catch (Exception e)
```
Comment (elliotritchie) [inline src/SFA.DAS.EAS.Web/Controllers/ActivitiesController.cs]: Is it worth logging `e`?

File: src/SFA.DAS.EAS.Web/Controllers/ActivitiesController.cs [hunk 2]
```diff
@@ -34,10 +35,17 @@ public ActivitiesController(IMapper mapper, IMediator mediator)
         [Route("latest")]
         public ActionResult Latest(GetLatestActivitiesQuery query)
         {
-            var response = _mediator.Send(query);
-            var model = _mapper.Map<LatestActivitiesViewModel>(response);
+            try
```
Comment (elliotritchie) [inline src/SFA.DAS.EAS.Web/Controllers/ActivitiesController.cs]: Is it worth adding a unit test for the try catch logic?

File: src/SFA.DAS.EAS.Web/Controllers/ActivitiesController.cs (hunk 1 above)
Comment (iangledhill) [reply src/SFA.DAS.EAS.Web/Controllers/ActivitiesController.cs]: Added

File: src/SFA.DAS.EAS.Web/Controllers/ActivitiesController.cs (hunk 2 above)
Comment (iangledhill) [reply src/SFA.DAS.EAS.Web/Controllers/ActivitiesController.cs]: Done

**Pull request merged**